public/app/features/alerting/unified/components/rules/central-state-history/CentralAlertHistory.tsx

```diff
@@ -31,7 +31,7 @@ import {
 } from 'app/types/unified-alerting-dto';
 
 import { stateHistoryApi } from '../../../api/stateHistoryApi';
-import { GRAFANA_DATASOURCE_NAME } from '../../../utils/datasource';
+import { GRAFANA_RULES_SOURCE_NAME } from '../../../utils/datasource';
```

Comment:
Using this instead of the other one, as the alert rule URLs were including `-- Grafana --`, and while they worked, it'd be nicer to have them the same as the main `view` URLs

Reply:
💯